experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1483,15 +1483,23 @@ def _validate_versions(self, data):
             )
 
         if (
-            application == NimbusExperiment.Application.DESKTOP
+            (
+                application != NimbusExperiment.Application.KLAR_ANDROID
+                and application != NimbusExperiment.Application.KLAR_IOS
+            )
             and is_rollout
             and parsed_min_version
             < NimbusExperiment.Version.parse(
-                NimbusConstants.DESKTOP_ROLLOUT_MIN_SUPPORTED_VERSION
+                NimbusConstants.ROLLOUT_MIN_SUPPORTED_VERSION[application]
             )
         ):
             self.warnings["firefox_min_version"] = [
-                NimbusConstants.ERROR_DESKTOP_ROLLOUT_VERSION
+                NimbusConstants.ERROR_ROLLOUT_VERSION.format(
```

Comment:
Do we want this to be an error or a warning? I'm thinking that we might want to change this to--
```py
raise serializers.ValidationError(
                {
                    "firefox_min_version": [
                        NimbusExperiment.ERROR_ROLLOUT_VERSION.format(...)
                    ],
                }
            )
```